tools/run_tests/run_tests.py

```diff
@@ -884,6 +884,38 @@ def runs_per_test_type(arg_str):
         msg = '\'{}\' is not a positive integer or \'inf\''.format(arg_str)
         raise argparse.ArgumentTypeError(msg)
 
+_COMPILER_CHOICES = ['default',
+                     'gcc4.4', 'gcc4.6', 'gcc4.9', 'gcc5.3',
+                     'clang3.4', 'clang3.5', 'clang3.6', 'clang3.7',
+                     'vs2010', 'vs2013', 'vs2015',
+                     'python2.7', 'python3.4',
+                     'node0.12', 'node4', 'node5',
+                     'coreclr'],
+
+
+def is_travis_enabled():
+  for arg in sys.argv:
+    if arg == "--travis" or arg == "-t":
+      return True
+  return False
+
+
+def compiler_choices(compiler):
```

Comment:
Add an octothorpe-delimited implementation comment explaining what this is, why it exists, and `TODO(https://github.com/grpc/grpc/issues/<number>):` hyperlinking to the issue filed that, when resolved, would obviate the need for it.